include/tesseract/renderer.h

```diff
@@ -151,6 +183,8 @@ class TESS_API TessResultRenderer {
   const char *file_extension_; // standard extension for generated output
   std::string title_;          // title of document being rendered
   int imagenum_;               // index of last image added
+  Pix *rendering_image_;       // Image to render with the extracted text
+  int rendering_dpi_;          // Resolution of the rendering_image
```

Comment:
Those new member variables and the related methods are currently only used by the PDF renderer.

Could they also be useful for other renderers in the future?

If not: what about moving them to TessPDFRenderer?